### DISCUSSION TITLE
Snowflake/Snowpark integration

### DISCUSSION THREAD

**Discussion by PierreFaraut** [marimo-team/marimo#2430] [Q&A]
Hi community! 
I was wondering how to leverage the SQL cell system to directly target snowflake without retrieving first everything as a dataframe (meaning, when doing a query, using snowflake engine instead of local engine). 
Any idea if we can already do that or how we could implement that? 
Thanks for your help!

**Comment by akshayka** [comment]
This isn't possible today, but we might one day support something like it. Thanks for the suggestion.
